common/kaitai/custom_decoder.h

```diff
@@ -0,0 +1,16 @@
+#ifndef KAITAI_CUSTOM_DECODER_H
+#define KAITAI_CUSTOM_DECODER_H
+
+#include <string>
+
+namespace kaitai {
+
+class custom_decoder {
```

Comment:
Is this used? I think this was the draft for the istream thing I posted earlier.

Reply:
No, the whole folder is a verbatim copy of https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/tree/master/kaitai
It can start being used in the future, if we will be using custom decoders.